app/assets/javascripts/ideabox_service.js

```diff
@@ -37,7 +37,7 @@ $('#index').delegate("#delete-idea", 'click', function(){
   var idea = this.closest('#idea')
     $.ajax({
       type: 'DELETE',
-       url: 'http://socalbros:3000/api/v1/ideas/'+ $(idea).attr('data-id'),
+       url: '/api/v1/ideas/'+ $(idea).attr('data-id'),
```

Comment:
If you DRY this code up a little bit, you can avoid some of the repetition when it comes to changing it in 7 places!